sa/sa.go

```diff
@@ -1565,6 +1565,27 @@ func (ssa *SQLStorageAuthority) GetAuthorizations(ctx context.Context, req *sapb
 	for name, a := range pendingAuthz {
 		authzMap[name] = a
 	}
+
+	// Fetch each of the authorizations' associated challenges
```

Comment:
Let's wrap this in a feature flag just in cast it causes a performance regression.

Reply:
Done in 52e23de. I debated making a separate feature flag for this with a more descriptive name but overall it felt more complicated since then the Wildcard issuance path has to check that both flags are enabled or it will break authz reuse for Wildcards because the chals won't be populated.